docker-entrypoint

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -il
+# copied from condaforge/linux-anvil
+
+# Enable the compiler toolset.
+#
+# Must do this before activating conda or
+# the system Python interpreter is not used.
+# This is a problem for many reasons. One being
+# that the Python scripts used to activate the
+# toolset are not Python 3 compatible.
+. scl_source enable devtoolset-2
```

Comment:
To verify that this works, I think you should add a test case in our test suite that builds a recipe which fails if gcc from devtoolset-2 is not present.

Reply:
Yes, was going to do that. I need to run the image first (haven't done so, yet) to see what the best way to ensure that would be.

Reply:
The added test case checks for `gcc (GCC) 4.8.2 20140120 (Red Hat 4.8.2-15)` exactly. This might be overly specific, but this way we'll hopefully notice unintended changes right away.